second-analysis-steps/rerun-stripping.md

```diff
@@ -20,6 +20,14 @@ This is nice, because most tools expect to read the stripping decisions from the
 
 Take a look at the file and try to find out what has changed compared to the [minimal DaVinci example](code/09-minimal-dv/ntuple_options.py).
 
+{% callout "Downloading the file during a Impactkit lesson" %}
+Lots of people downloading the same file at the same time can be very slow.
+The file in the minimal DaVinci example can be copied using this command:
+```bash
+$ xrdcp root://eosuser.cern.ch//eos/user/m/mhilton/impact-kit/00070793_00000001_7.AllStreams.dst .
```

Comment:
Would it not make sense to have "StarterKit files" such as this one centrally in one EOS StarterKit WG location to create, just as physics working groups have attributed locations? Seems for sure ideal in the longer run.

Reply:
We have a service account (`lhcbsk`) which I can copy it to tomorrow.

Reply:
Yes this would make sense in the long run for sure - I was just making a quick fix for this year's Impactkit to make sure all the lessons can be run.

Reply:
Understood. See my comment as contructive, forward-looking ;-). (Meaning I take your point, of course.)

Reply:
```suggestion
$ xrdcp root://eosuser.cern.ch//eos/user/l/lhcbsk/impactkit/00070793_00000001_7.AllStreams.dst .
```